File: PYTHON💕/zadanie_3.py
input_numbers = input().split()
output_indices = []
for index, number in enumerate(input_numbers):
    digits = [int(digit) for digit in number]
    sum_digits = sum(digits)
    product_digits = 1
    for digit in digits:
        product_digits *= digit
    if sum_digits < product_digits:
        output_indices.append(index)
print(" ".join(map(str, output_indices)))

File: JAVA SCRIPT/zadanie_3.js
const input = prompt().split(" ");
const output = [];
for (let i = 0; i < input.length; i++) {
    const number = input[i];
    const digits = number.split("").map(Number);
    const sum = digits.reduce((a, b) => a + b, 0);
    const product = digits.reduce((a, b) => a * b, 1);
    if (sum < product) {
        output.push(i);
    }
}
console.log(output.join(" "));
